EngineLayer/PeptideSpectralMatch.cs

```diff
@@ -218,37 +225,60 @@ public void ResolveAllAmbiguities()
             MatchedFragmentIons = PeptidesToMatchingFragments.First().Value;
         }
 
-        public int GetLongestIonSeriesBidirectional(PeptideWithSetModifications peptide)
+        public int GetLongestIonSeriesBidirectional(Dictionary<PeptideWithSetModifications, List<MatchedFragmentIon>> PeptidesToMatchingFragments, PeptideWithSetModifications peptide)
```

Comment:
For both this method and "GetCountComplementaryIons", you don't need the first input parameter because it's an object within the class. Otherwise this could just be a static method.

Reply:
made static